src/cbl-mariner/manifest.json

```diff
@@ -42,6 +42,127 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/cbl-mariner/2.0/crossdeps",
+              "os": "linux",
+              "osVersion": "cbl-mariner2.0",
+              "tags": {
+                "cbl-mariner-2.0-crossdeps-local": {
+                  "isLocal": true
+                }
+              }
+            }
+          ]
+        },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/cbl-mariner/2.0/crossdeps-amd64",
+              "os": "linux",
+              "osVersion": "cbl-mariner2.0",
+              "tags": {
+                "cbl-mariner-2.0-crossdeps-amd64-local": {
+                  "isLocal": true
+                }
+              }
+            }
+          ]
+        },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/cbl-mariner/2.0/crossdeps-arm",
+              "os": "linux",
+              "osVersion": "cbl-mariner2.0",
+              "tags": {
+                "cbl-mariner-2.0-crossdeps-arm-local": {
+                  "isLocal": true
+                }
+              }
+            }
+          ]
+        },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/cbl-mariner/2.0/crossdeps-artifacts",
+              "os": "linux",
+              "osVersion": "cbl-mariner2.0",
+              "tags": {
+                "cbl-mariner-2.0-crossdeps-artifacts-local": {
+                  "isLocal": true
+                }
+              }
+            }
+          ]
+        },
```

Comment:
The Dockerfiles in the manifest should be listed in dependency order. So this one should be listed first in your changes.